README.md

```diff
@@ -1,134 +1,16 @@
-
-    You can check the progress of the model loading by looking at the error logs for the SLURM job, which should have lines like
-
-    ```
-    Loading safetensors checkpoint shards:   0% Completed | 0/191 [00:00<?, ?it/s]
-
-    Loading safetensors checkpoint shards:   1% Completed | 1/191 [00:04<13:59,  4.42s/it]
-
-    Loading safetensors checkpoint shards:   1% Completed | 2/191 [00:09<15:58,  5.07s/it]
-
-    Loading safetensors checkpoint shards:   2% Completed | 3/191 [00:26<32:54, 10.50s/it]
-    ```
-
-    When the model is fully loaded and the server is ready to handle requests, you should see lines like
-
-    ```
-    INFO:     Started server process [2405764]
-    INFO:     Waiting for application startup.
-    INFO:     Application startup complete.
-    INFO:     Uvicorn running on http://0.0.0.0:8000 (Press CTRL+C to quit)
-    ```
-
-## Using the vLLM Server
-
-The vLLM server will be set up on the first node of the SLURM job.
-This is marked in the head node in the SLURM job logs through a line similar to,
-
-```
-Head node: holygpu8a15303
-```
-
-but can also be found by using `squeue` to look at the nodes provisioned for the job and taking the first one. You can then ssh into this gpu node. Following the example above, if the first node is `holygpu8a15303`, we can run the following command to enter the gpu node.
-
-```bash
-ssh holygpu8a15303
-```
-
-The server will then be running on `localhost:8000`. You can send HTTP requests to the `/v1/completions` endpoint to run the model on your prompts.
-
-```
-curl http://localhost:8000/v1/completions \
-    -X POST \
-    -H "Content-Type: application/json" \
-    -d '{
-        "model": "/n/holylfs06/LABS/kempner_shared/Everyone/testbed/models/Llama-3.1-405B",
-        "prompt": "San Francisco is a",
-        "max_tokens": 7,
-        "temperature": 0
-    }'
-```
-Note that the model field for the JSON needs to be the directory of the model being served. vLLM uses the model directory to identify the model, so you should use the following strings for the `model` field.
-
-- Llama 3.1 70B: `/n/holylfs06/LABS/kempner_shared/Everyone/testbed/models/Llama-3.1-70B`
-- Llama 3.1 405B: `/n/holylfs06/LABS/kempner_shared/Everyone/testbed/models/Llama-3.1-405B`
-
-For Python applications, you can use the `requests` library to send your HTTP requests.
-
-```python
-import requests
-
-response = requests.post('http://localhost:8000/v1/completions', json = {
-    "model": "/n/holylfs06/LABS/kempner_shared/Everyone/testbed/models/Llama-3.1-405B",
-    "prompt": "San Francisco is a",
-    "max_tokens": 500,
-    "temperature": 0
-})
-
-output = response.json()
-```
-Additional arguments like `top_k` and `min_p` are also available for adjusting how tokens are sampled. See the [vLLM docs](https://docs.vllm.ai/en/latest/dev/sampling_params.html) for the available arguments.
+> **Note:** Follow this repository for regular updates on deployment instructions for the latest models on AI clusters.
```

Comment:
Note does not render properly, please double check. 

https://github.com/orgs/community/discussions/16925